component/ocpscc.jsonnet

```diff
@@ -0,0 +1,65 @@
+// instance-specific security context constraint object for openshift
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local inv = kap.inventory();
+// The hiera parameters for the component
+local params = inv.parameters.cert_exporter;
+
+local isOpenshift = std.startsWith(inv.parameters.facts.distribution, 'openshift');
+
+local openshiftScc = {
```

Comment:
why does this tool need a custom SCC on OpenShift?

Reply:
The tool needs some special privileges to scrape the secrets in all namespaces and to run on the nodes to scrape the file based certificates.
I did not find a predefined scc matching its requirements.